market_next_door_api/market_next_door_api/serializers.py

```diff
@@ -34,4 +34,23 @@ def to_representation(self, instance):
 class VendorSerializer(serializers.ModelSerializer):
   class Meta:
     model = Vendor
-    fields = ['id', 'market', 'vendor_name', 'first_name', 'last_name', 'email', 'location']
\ No newline at end of file
+    fields = ['id', 'market', 'vendor_name', 'first_name', 'last_name', 'email', 'location']
+
+# preorders
+class PreorderSerializer(serializers.ModelSerializer):
+  class Meta:
+    model = Preorder
+    fields = ['id', 'customer', 'item', 'ready']
+  
+  def to_representation(self, instance):
+    representation = super().to_representation(instance)
+    data = {
+      "id": representation['id'],
+      "type": "preorder",
+      "attributes": {
+        "customer": representation['customer'],
+        "item": representation['item'],
+        "ready": representation['ready']
+      }
+    }
+    return data
```

Comment:
We'll want to delete this section for the serializer. We agreed earlier that we'll just give FE the simplest version of a response as we can generate (without extra formatting). So we just need lines 39 - 43 now.

Reply:
Agreed, let's simplify the serialization.